### ISSUE TITLE
User sign_up failing in production

### ISSUE THREAD

**Issue by felipefreitag** [felipefreitag/pangeas#88]
It's failing with `ActiveModel::UnknownAttributeError (unknown attribute 'affiliate_tag' for User.):` even though the corresponding migration has been run. 

```
2017-11-13T16:57:43.539089+00:00 app[web.1]: I, [2017-11-13T16:57:43.539033 #4]  INFO -- : [ead517cd-3793-4c34-928c-e8da7580fbeb]   Parameters: {"utf8"=>"✓", "authenticity_token"=>"50fmk6dOaf95dzqlx2HxVMgwPZcyha0I3WG4qoYVndnyIKhnYV5iQQddzFesPSf9UbebKRt6PQOY/SIo4hk7Xg==", "recurrence"=>"monthly", "user"=>{"affiliate_tag"=>"4", "first_name"=>"foobar", "last_name"=>"barfoo", "email"=>"foobarcom@mailinator.com", "password"=>"[FILTERED]", "password_confirmation"=>"[FILTERED]"}, "commit"=>"Inscrever-se"}
2017-11-13T16:57:43.684066+00:00 app[web.1]: I, [2017-11-13T16:57:43.683971 #4]  INFO -- : [ead517cd-3793-4c34-928c-e8da7580fbeb] Completed 500 Internal Server Error in 145ms (ActiveRecord: 0.0ms)
2017-11-13T16:57:43.685082+00:00 app[web.1]: F, [2017-11-13T16:57:43.685022 #4] FATAL -- : [ead517cd-3793-4c34-928c-e8da7580fbeb]   
2017-11-13T16:57:43.685155+00:00 app[web.1]: F, [2017-11-13T16:57:43.685102 #4] FATAL -- : [ead517cd-3793-4c34-928c-e8da7580fbeb] ActiveModel::UnknownAttributeError (unknown attribute 'affiliate_tag' for User.):
```

When running 
```
heroku run rails c
User.first
```
The output is
`=> #<User id: 2, email: "foo@bar.com", first_name: "foo", last_name: "bar", admin: false, created_at: "2017-11-08 17:59:06", updated_at: "2017-11-13 16:39:55", iugu_id: "D55BECCBAF6446CF9BB564791313EC9C", affiliate_tag: nil>`

In other words, the corresponding `affiliate_tag` field does exist. It's also possible to create a new user from the console.

**Comment by felipefreitag** [comment]
Now it's working. o.O
